content/chronograf/v1.10/about_the_project/release-notes-changelog.md

```diff
@@ -8,6 +8,38 @@ menu:
     parent: About the project
 ---
 
+## v1.10 [TBD]
+
+### Features
+
+- Update the admin UI to provide database-level role-based access control (RBAC). The UI also shows permissions mappings more intuitively and includes:
+  - improved user creation
+  - improved role creation
+  - improved visualization of effective permissions on `Users` and `Roles` pages
+- Add [`reader role`](/chronograf/v1.10/administration/managing-chronograf-users/#readers-rolereader) for users that require read-only access to dashboards.
```

Comment:
is this same as viewer role? I switched into the main chrono-1.10 branch and didn't see this header #readers-rolereader, so want to double check this is correct link?